doc/decisions/0001-context-in-translations.md

```diff
@@ -0,0 +1,84 @@
+  | `$..add_block.cancel_text` | List collector add block cancel link | |
+  | `$..content.title` | Content page heading | The main heading on a Content page |
+  | `$..content.contents.[*].title` | Content page contents title | |
+  | `$..content.contents.[*].description` | Content page contents description | |
+  | `$..content.contents.[*].list` | Content page contents list | |
+  | `$..question.title` | Question text | |
+  | `$..question.description` | Question description | |
+  | `$..question.instruction` | Question instruction | Instructions for field interviewers |
+  | `$..question.definitions.[*].title` | Question definition link | |
+  | `$..question.definitions.[*].contents.[*].title` | Question definition title | |
+  | `$..question.definitions.[*].contents.[*].description` | Question definition description | |
+  | `$..question.definitions.[*].contents.[*].list` | Question definition list  | |
+  | `$..question.guidance.contents.[*].title` | Question guidance title | |
+  | `$..question.guidance.contents.[*].description` | Question guidance description | |
+  | `$..question.guidance.contents.[*].list` | Question guidance list | |
+  | `$..answers.[*].validation.messages` | Answer error message | |
+  | `$..answers.[*].label` | Answer label | |
+  | `$..answers.[*].description` | Answer description | |
+  | `$..answers.[*].playback` | Relationships playback template | Used for playback before a relationship is chosen e.g. "{person2} is {person1}'s ..." |
+  | `$..answers.[*].options.[*].label` | Answer option | |
+  | `$..answers.[*].options.[*].description` | Answer option description | |
+  | `$..answers.[*].options.[*].detail_answer.label` | Detail answer label | |
+  | `$..answers.[*].options.[*].detail_answer.description` | Detail answer description | |
+  | `$..answers.[*].options.[*].title` | Relationships answer option question text | Question text to set when this answer option is selected e.g. "Thinking about {person1}, {person2} is their husband or wife"|
+  | `$..answers.[*].options.[*].playback` | Relationships answer option playback text | Used for playback when a relationship is chosen e.g. "{person2} is {person1}'s husband or wife" |
+  | `$..answers.[*].guidance.show_guidance` | Answer guidance show link | |
+  | `$..answers.[*].guidance.hide_guidance` | Answer guidance hide link | |
+  | `$..answers.[*].guidance.contents.[*].title` | Answer guidance title | |
+  | `$..answers.[*].guidance.contents.[*].description` | Answer guidance description |
+  | `$..answers.[*].guidance.contents.[*].list` | Answer guidance list | |
+
+- Where helpful to reference another property value, an additonal comment should be added. Initially this will be:
+
+  | JSON Path | Comment |
+  |-----------|---------|
+  | `$..answers.[*].description`                           | For answer label: {answer label} |
```

Comment:
This is the only 1 that confused me. I don't quite understand why is this `For answer label`?

Reply:
- The description is associated with the answer label
- The `[*]` in the JSON path is a wildcard match that matches each answer array object. I think using `*` would match any `description` property at an unlimited level of nesting under answers, which is a wider match than is needed.

Reply:
Regarding the wildcard comment.

Using `$.sections.[*].title` matches all titles under section key and its child elements.

To get the title element of just the section key, we need either:
- `$.sections[*].title` or `$.sections.*.title`

I used https://jsonpath.com/ to test this.

Reply:
My bad. I used https://github.com/kennknowles/python-jsonpath-rw to test the JSON path and it seems to work with the incorrect syntax. I will update all of the `.[*]` to `[*]`.